web/src/api/index.ts

```diff
@@ -1,7 +1,37 @@
 import createClient from 'openapi-fetch';
 import { components, paths } from './schema';
+import { createLocalStorageSignal } from '../utils/localstorage';
 
 const client = createClient<paths>({ baseUrl: '/api' });
+const [currentSessionToken, setCurrentSessionToken] =
```

Comment:
It might be worth to start splitting different API calls into different files, e.g. `api/auth` and `api/stacklets`

Reply:
3c8d5d3bf61eba8e97e57cb2b83887efaf8f03ad